### PR TITLE
User can view history of time-recordings

### PR DESCRIPTION
**Pull request by yanqiuzhang**
#### Description

_As a User
In order to view my work history
I would like to see my records displayed_

Changes proposed in this pull request:
* Config send request from api
* Add styling 
* Fetch data from api
* Display results on time tracking page

### CODE REVIEW

File: cypress/integration/userCanViewTimeHistory.feature.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		cy.server();
+		cy.route({
+			method: 'GET',
+			url: 'https://demo-stable.kimai.org/api/timesheets',
+			response: 'fixtures:save_data.json',
+			headers: {
+				"X-AUTH-USER": "susan_super",
+				"X-AUTH-TOKEN": "api_kitten"
+			}
+		})
```
Comment (tochman) [inline cypress/integration/userCanViewTimeHistory.feature.js]: Why are you using this response?

File: cypress/integration/userCanViewTimeHistory.feature.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		cy.get('input[type=password]').type('api_kitten')
+		cy.get('button[type=submit]').click()
+		cy.get('input[name="begin"]').type("07:00")
+		cy.get('input[name="end"]').type("08:00")
+		cy.get('button[name="create"]').click()
+		cy.contains('Your time was saved')
+		cy.contains('07:00')
+		cy.contains('08:00')
+	})
+});
```
Comment (tochman) [inline cypress/integration/userCanViewTimeHistory.feature.js:48]: I can't see how this test relates to the current user story. Please explain

Comment (NiciP) [reply cypress/integration/userCanViewTimeHistory.feature.js]: we would like to show the time recordings displayed on the page after we submitted them. we can elaborate on the test once we have info displayed
image

Comment (NiciP) [reply cypress/integration/userCanViewTimeHistory.feature.js:48]: we got clarity on this, agree. will rectify to only focus on testing for fetching the info

File: cypress/integration/userCanViewTimeHistory.feature.js (hunk 1 above)
Comment (NiciP) [reply cypress/integration/userCanViewTimeHistory.feature.js]: created a new get_data.json file with that response

**Review by oliverochman — APPROVED** [review]

**Pull request merged**